README.md

```diff
@@ -1 +1,240 @@
+ ### Get into development
+ 
+ In the terminal, run these commands to get into development.
+ ```
+ $ cd cozy-homes-front-end
+ 
+ $ yarn install (install dependencies)
+ ```
+ **Note: You need to start the `rails server` first to ensure proper use of the frontend. So make sure you have followed the instructions available on [cozy-homes-back-end](https://github.com/HunterX-7/cozy-homes-back-end.git).**
+ ```
+ $ yarn start
+ ```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ <!-- Kanban Board -->
+ 
+ ## 📃 Kanban Board <a name="kanban-board"></a>
+
+ The Kanban Board made for this project can be found [here](https://github.com/users/HunterX-7/projects/2).
+
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+ <!-- AUTHORS -->
+ 
+ ## 👥 Authors <a name="authors"></a>
+
+ 👤 *Francisco Quintero*
+
+- GitHub: [@HunterX-7](https://github.com/HunterX-7)
+- Twitter: [@franquin07](https://twitter.com/franquin07)
+- LinkedIn: [Francisco Quintero](https://www.linkedin.com/in/francisco-asis-quintero-cede%C3%B1o/)
+
+ 
+👤 *Pierre Barba*
+
+- GitHub: [@pierreBarba](https://github.com/PierreBarba)
+- Twitter: [@pierreBarba](https://twitter.com/home)
+- LinkedIn: [@pierreBarba](https://www.linkedin.com/in/jean-pierre-barba/)
+
+ 
+👤 *Juan Pablo Guerrero*
+
+- GitHub: [@Juan Pablo Guerrero](https://github.com/Juanpa8830)
+- LinkedIn: [@Juan Pablo Guerrero](https://www.linkedin.com/in/juanpabloguerrerorios/)
+
+
+👤 *Gabriel Cilfone*
+
+- GitHub: [Gabriel Cilfone](https://github.com/cilfonegabriel)
+- LinkedIn: [Gabriel Cilfone](www.linkedin.com/in/gabriel-cilfone/)
+ 
+ 
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+ ## 🔭 Future Features <a name="future-features"></a>
+ 
+ - [ ] *Add animation to create smooth user experience.*
+ - [ ] *Enable only admin users to add or delete houses.*
+ - [ ] *Implement proper user authentication from the front-end to the server.*
+ 
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+ <!-- CONTRIBUTING -->
+ 
+ ## 🤝 Contributing <a name="contributing"></a>
+ 
+ Contributions, issues, and feature requests are welcome! Add suggestions by opening new issues.
+ 
+ Feel free to check the [issues page](https://github.com/HunterX-7/cozy-homes-front-end/issues).
+ 
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+ <!-- SUPPORT -->
+ 
+ ## ⭐️ Show your support <a name="support"></a>
+ 
+ Give a ⭐️ if you like this project!
+ 
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+ <!-- ACKNOWLEDGEMENTS -->
+ 
+ ## 🙏 Acknowledgments <a name="acknowledgements"></a>
+ 
+ Authors would like to thank:
+ - [Murat Korkmaz](https://www.behance.net/muratk) for the Original Design Idea.
+ - [Microverse](https://www.microverse.org/)
+ - Code Reviewers
+ - Coding Partners
+ 
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+ <!-- LICENSE -->
+ 
+ ## 📝 License <a name="license"></a>
+ 
+ This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ]  On the "License" section of your readme, the link is broken because it points to a file that does not exist in your project.

## Tips and extra help

Go [here](https://github.com/microverseinc/readme-template), grab the `MIT.md` file from that repo, paste it on the root of your project, and link to it in the "License" section of your readme.

That way, your app will be well documented 💪